cEP-0018.md

```diff
@@ -22,6 +22,38 @@ they can write advanced linting bears. This will aim at supporting the more
 flexible visitor based method of AST traversal (as opposed to listener based
 mechanisms).
 
+ANTLR has a `HiddenTokenStream` which recieves all the tokens irrelevant to the
+grammar such as whitespaces. These are not passed on to the parser and as a
+result are not returned when simply fetching text from a node in parse tree.
+
+Consider the code:
+
+```python
+var_x = 5
+if var_x:
+    print('Variable x is', var_x)
+else:
+    print('Variable x is not set')
```

Comment:
use `"` here to show whether the `"`/`'` is lost or kept by ANTLR

Reply:
Just for a textual note: depending on the parser context rules / lexer context rules defined in the grammar, the `"`/`'` may or may not be lost. 
w.r.t the python grammar, the `"` is not lost.